test/signalrclient-e2e-tests/url.h

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+extern utility::string_t url;
```

Comment:
No, complier returns error like: 
"class std::basic_string<wchar_t,struct std::char_traits<wchar_t>,class std::allocator<wchar_t> > url" (?url@@3V?$basic_string@_WU?$char_traits@_W@std@@V?$allocator@_W@2@@std@@A) already defined in connection_tests.obj